### PR TITLE
Update package READMEs

### PR DESCRIPTION
**Pull request by CiaranMn**
## 🌟 What is the purpose of this PR?

Some basic updates to package READMEs to bring them to up to date for the `0.3` spec.

[Asana task](https://app.asana.com/0/1203543026679380/1203855800580752/f) (internal)

### CODE REVIEW

File: libs/block-template-custom-element/README.md [hunk 1]
```diff
@@ -8,11 +8,38 @@ See [https://blockprotocol.org/docs/developing-blocks](https://blockprotocol.org
...
+- change the default `tagName` under `blockType` to the tag for your element
+- optionally update additional fields which will be used to identify and describe the block when published:
+  - `displayName`: a friendly display name
+  - `examples`: an array of example data structures your block would accept and use
+  - `image`: a preview image showing your block in action (in place of `public/block-preview.png`)
+  - `icon`: an icon to be associated with your block (in place of `public/omega.svg`)
+  - `name`: a slugified name for your block (which may differ to the package `name` in package.json)
+    - this may either be in the format `slug` or `@namespace/slug` where `namespace` is your blockprotocol.org username
+
+1.  Run `yarn schema` to automatically TypeScript types from your block's entity type / `schema`
```
Comment (luisbettencourt) [inline libs/block-template-custom-element/README.md:38]: to automatically generate, right?

File: libs/block-template-react/README.md [hunk 2]
```diff
@@ -8,7 +8,35 @@ See https://blockprotocol.org/docs/developing-blocks
...
+- set `schema` to the URL of the entity type you created in the previous step
+- optionally update additional fields which will be used to identify and describe the block when published:
+  - `displayName`: a friendly display name
+  - `examples`: an array of example data structures your block would accept and use
+  - `image`: a preview image showing your block in action (in place of `public/block-preview.png`)
+  - `icon`: an icon to be associated with your block (in place of `public/omega.svg`)
+  - `name`: a slugified name for your block (which may differ to the package `name` in package.json)
+    - this may either be in the format `slug` or `@namespace/slug` where `namespace` is your blockprotocol.org username
+
+1.  Run `yarn schema` to automatically TypeScript types from your block's entity type / `schema`
```
Comment (luisbettencourt) [inline libs/block-template-react/README.md:37]: Same thing here

**Review by luisbettencourt — CHANGES_REQUESTED** [review]

File: libs/block-template-custom-element/README.md (hunk 1 above)
Comment (CiaranMn) [reply libs/block-template-custom-element/README.md]: Yep, thanks! https://github.com/blockprotocol/blockprotocol/pull/986/commits/f9f0a257aea9593775afa9d72b41ba01ad561530

File: libs/block-template-react/README.md (hunk 2 above)
Comment (CiaranMn) [reply libs/block-template-react/README.md]: https://github.com/blockprotocol/blockprotocol/pull/986/commits/f9f0a257aea9593775afa9d72b41ba01ad561530

**Review by luisbettencourt — APPROVED** [review]

**Pull request merged**